tooling/templatize/pkg/pipeline/run.go

```diff
@@ -481,15 +482,28 @@ func executeNode(logger logr.Logger, executor Executor, graphCtx *graph.Graph, n
 	var output Output
```

Comment:
You probably want to make sure that timing metadata, etc, correctly shows re-tries

Reply:
Did not think about that, of course... 
Will look into that